src/main/kotlin/io/github/mojira/arisa/infrastructure/config/ArisaConfig.kt

```diff
@@ -166,6 +166,11 @@ object Arisa : ConfigSpec() {
         }
 
         object Language : ModuleConfigSpec() {
+            val apiQuotaWarningThreshold by optional<Double?>(
+                description = "When the number of remaining API quota units becomes equal to or lower than this" +
+                        " value a warning will be logged.",
+                default = null
```

Comment:
Let's set it to 10 by default? (That should be around 100 tickets

Reply:
Sure! Also feel free to squash these commits when merging.

Reply:
Ah, sorry misunderstood your comment. Do you mean change the default here?
I thought it might be useful to allow disabling the logging (for whatever reason) by simply not specifying a value in the config.
Do you think the threshold should be non-`null` then?

Reply:
Yep